### DISCUSSION TITLE
Connection between IP Ranges and Prefixes

### DISCUSSION THREAD

**Discussion by Zombie-Toad** [netbox-community/netbox#18187] [Q&A] [answered]
Hey,

I would like to know if there is a specific reason, why I don't see my IP Ranges in Prefix details?
Let's say I have a Prefix for 192.168.0.0/24 and a DHCP Pool range of 192.168.0.100-192.168.0.254 defined as IP Range.

I would assume, that I can only use the IP addressess form 192.168.0.[1-99].
In the Prefix I can see that the utilization  is at 61% but I could use all IP addresses inside the prefix and potentially run into duplicate IP address errors.

Prefix:
![image](https://github.com/user-attachments/assets/505d8d09-08c2-4ed6-bd28-e8bcbb6fe8d6)

IP Range (the range is also marked as utilized):
![image](https://github.com/user-attachments/assets/ea442d23-af9a-4131-8e5c-0129a27dc5d0)

Prefix details (with available IPs at 99):
![image](https://github.com/user-attachments/assets/3ff065d1-dbf0-4e30-92f1-c67d72f3fb4d)

Available IP addresses in Prefix:
![image](https://github.com/user-attachments/assets/55b05ec5-2a80-4964-a1f2-37ad14ed849d)

**Comment by chris240189** [comment, marked as answer]
same as https://github.com/netbox-community/netbox/discussions/17996#discussioncomment-11232651

**Reply by Zombie-Toad** [reply]
Thanks for the hint, I keep an eye on the feature request mentioned by candlerb (https://github.com/netbox-community/netbox/issues/9763).
